### PR TITLE
CLOUDP-47509: push tag for each image repo (latest & non-latest)

### PR DESCRIPTION
**Pull request by mrgunior**
Noticeable changes:
- Adjusted regex (sed) for only removing the first v we encounter (if any) and keep the rest
- Script for building a single image, and pushing two different versions to the docker repo. This of course does depend on how its tagged. If a commit is tagged as `v0.0.1`, this will push two images to the docker repo. One with `latest` and one with `0.0.1` tag. Any other tagging format indicates not a latest and only 1 image is created and pushed to the docker repo.

### CODE REVIEW

File: .evergreen.yaml [hunk 1]
```diff
@@ -64,7 +65,7 @@ functions:
       params:
         working_dir: src/atlas-service-broker
         script: |
-          export GITHUB_TOKEN=${github_token}
+          export GITHUB_TOKEN=${github}
```
Comment (BenElgar) [inline .evergreen.yaml]: What is this change?

File: .evergreen.yaml [hunk 2]
```diff
@@ -88,7 +89,12 @@ functions:
         working_dir: src/atlas-service-broker
         script: |
           docker login --username ${docker_username} --password ${docker_password} ${docker_repo}
+
           docker push ${docker_repo}/${docker_name}:${version}
+          if [[ ${version} =~ [0-9]+.[0-9]+$ ]]; then
```
Comment (BenElgar) [inline .evergreen.yaml]: This should include the third stanza of the version number (i.e. the patch version).

File: .evergreen.yaml (hunk 1 above)
Comment (mrgunior) [reply .evergreen.yaml]: yeah it should be github_token.

File: .evergreen.yaml (hunk 2 above)
Comment (mrgunior) [reply .evergreen.yaml]: cool, np.

**Pull request merged**